CHANGELOG.md

```diff
@@ -32,6 +34,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   (#[1713](https://github.com/nix-rust/nix/pull/1713))
 - `nix::poll::ppoll`: `sigmask` parameter is now optional.
   (#[1739](https://github.com/nix-rust/nix/pull/1739))
+- `signal:SigSet` it's now marked as `repr(trasparent)`.
```

Comment:
There's a minor typo or two here, but we can clean it up in a later commit before the next release. No reason for this to stay open.